core/src/main/java/org/apache/iceberg/encryption/WrappedEncryptionKey.java

```diff
@@ -19,29 +19,24 @@
 package org.apache.iceberg.encryption;
 
 import java.io.Serializable;
+import java.nio.ByteBuffer;
 
-public class KeyEncryptionKey implements Serializable {
+public class WrappedEncryptionKey implements Serializable {
```

Comment:
I renamed this because I think it will be produced by `TableMetadataParser` and doesn't need to be specific to key encryption. I think we will just have a list of keys that are held by table metadata.